src/pages/Homepage.js

```diff
@@ -19,31 +36,53 @@ const Homepage = () => {
     }
   };
 
+  function updateForecastDegree(event){
+    setDegreeType({
+      degreeType: event.target.value
+    })
+    console.log(setDegreeType)
+  }
+
+  const fahrenheit = Math.round(weatherData.main === undefined ? 0 : weatherData.main.temp );
+  const celsius = Math.round((fahrenheit - 32)* 5/9);
+
   return (
     <div className="container">
-      <input 
+      <input style={{ fontFamily: 'Montserrat, sans-serif' }}
+      type = "search"
       className="input" 
       placeholder="Enter city..."
       onChange={e => setCity(e.target.value)}
       value={city}
       onKeyPress={getWeather} />
 
-      {typeof weatherData.main === 'undefined' ? (
-        <div>
-          <p>Welcome to weather app! Enter in a city to get the weather of.</p>
+      {weatherData.main === undefined ? (
+        <div className="welcomeContainer">
+          <p>Welcome to weather app! Enter in a city in the field above to get its weather.</p>
         </div>
       ) : (
         <div className="weather-data">
           <p className="city">{weatherData.name}</p>
-          <p className="temp">{Math.round(weatherData.main.temp)}ºF</p>
+          <div className="temp">
+            {/* <div>
+             <DegreeToggle updateForecastDegree={updateForecastDegree} degreeType={degreeType}/>
+            </div> */}
+            <div className="unitContainer">
+            {Math.round(weatherData.main.temp)}ºF / {Math.round(celsius)}&deg;C
+            {/* {degreeType === "celsius" ? celsius + "°C" : fahrenheit + "°F" } */}
+            </div>
+          </div>
+          <div>
+            {/* <img src={cloud} alt=""/> */}
+          </div>
           <p className="weather">{weatherData.weather[0].main}</p>
           <p></p>
         </div>
       )}
 
       {
         weatherData.cod ==='404' ? (
```

Comment:
this could be moved to a variable and used here for readability and may increase your code quality.